### ISSUE TITLE
Failed to format large request body

### ISSUE THREAD

**Issue by MichaelBui** [csarrazi/CsaGuzzleBundle#189]
I've got this error after upload big file (~10MB video) via Guzzle:

```
FatalErrorException in CurlFormatter.php line 137:
Error: escapeshellarg(): Argument exceeds the allowed length of 2097152 bytes
```

Traceback:
```
1. in CurlFormatter.php line 137
2. at escapeshellarg() in CurlFormatter.php line 137
3. at CurlFormatter->extractBodyArgument() in CurlFormatter.php line 220
4. at CurlFormatter->extractArguments() in CurlFormatter.php line 54
5. at CurlFormatter->format() in GuzzleCollector.php line 100
6. at GuzzleCollector->collect() in Profiler.php line 218
7. at Profiler->collect() in ProfilerListener.php line 128
8. at ProfilerListener->onKernelResponse() in WrappedListener.php line 61
```

Current dependency in our project: `dev-master`

**Comment by csarrazi** [comment]
For now, just disable the support for the curl formatter for now, by uninstalling namshi/cuzzle.

In the longer term, we should disable the curl formatter for large bodies (100kb), just like we trim the content for the profiler.

**Comment by Avariya** [comment]
The problem is in cuzzle. I created fix couple of monthes ago, but cuzzle ignoring it.
After that fix for csaguzzlebundle was created and it is already merged at dev branch

**Comment by MichaelBui** [comment]
I guess your changes are here https://github.com/namshi/cuzzle/pull/17/files

However as @csarrazi said, we should skip curl formatter for large contents

**Comment by Avariya** [comment]
Yes. Ok. I will try to pick this task next week if no one will assign it to himself first.

**Comment by csarrazi** [comment]
Yes, I know that the problem lies within cuzzle too. It makes no sense displaying a whole 10MB file in the profiler when uploading such a file, for both the profiler panel and the cuzzle output.

**Comment by csarrazi** [comment]
Fixed by #201
